src/io/jesi/backpack/collection.cljc

```diff
@@ -371,3 +371,19 @@
   [m]
   (= (count m)
      (count (set (vals m)))))
+
```

Comment:
wanted to create some helper methods for doing these filter operations, cause I find myself doing them quite often